### ISSUE TITLE
Question about the output latent of VAVAE-f16d32

### ISSUE THREAD

**Issue by DaiZhewei** [hustvl/LightningDiT#35]
Hi, author,

I found that the output latent of the pretrained VAVAE-f16d32 has a channel size of 64, which does not match the input size of 4 $\times$ 32 $\times$ 32 for the DiT.

Could there be an issue with my usage? If so, how should I resolve it? 

Thank you!

**Comment by JingfengYao** [comment]
Actually, it is the parameter of the distribution. Please refer to [here](https://github.com/hustvl/LightningDiT/blob/main/vavae/ldm/modules/distributions/distributions.py). So the latents after sampling are 16x16x16 (for 256x256x3 images). 32x32x4 is the input size of SD-VAE, and our setting is slightly different.

**Comment by JingfengYao** [comment]
``Plug-in`` is a description for VF loss. This means you can train or fine-tune VAEs by directly adding VF loss without any other schedule modification. However, if you want to replace SD-VAE with VAE at different f and d, it is a diffusion model architecture design and pre-training weight inheritance problem. It is not that ``plug-in``.

**Comment by YoungLer** [comment]
> I will try fine-tuning the SD-VAE with VF loss. Thank you very much for your constructive advice and help!我将尝试微调具有 VF 损失的 SD-VAE。非常感谢您的建设性建议和帮助！

Hi,
Have you tried fine-tuning the SD-VAE? May I ask does it work?
